src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -134,13 +144,20 @@ final class OnboardingInfoViewController: UIViewController {
 		present(alert, animated: true, completion: nil)
 	}
 
-	private func updateUI() {
+	private func updateUI(exposureManagerState: ExposureManagerState) {
 		guard isViewLoaded else { return }
 		guard let onboardingInfo = onboardingInfo else { return }
 
 		titleLabel.text = onboardingInfo.title
 
-		imageView.image = UIImage(named: onboardingInfo.imageName)
+		let exposureNotificationsNotSet = exposureManagerState.status == .unknown
```

Comment:
exposureManagerState.status == .unknown is false doesnt implicate exposureNotificationsNotSet ... See more details here: https://developer.apple.com/documentation/exposurenotification/enstatus?language=objc

Reply:
This point is resolved in a call. Just to summarise, to determine should two buttons be shown in the 3rd onboarding step, with user getting the chance to decide whether to accept it or not, or one button should be shown, along with the state view, and taking into the consideration that after the reset Exposure Manager is not activated yet, the only way to determine and differentiate this is ```exposureManagerState.status == .unknown```.